pyproject.toml

```diff
@@ -26,11 +26,11 @@ isort = "^5.10.1"
 mypy = "^0.991"
 
 [tool.black]
-line-length = 79
+line-length = 88
 skip-magic-trailing-comma = true
 
 [tool.ruff]
-line-length = 79
+line-length = 88
```

Comment:
both ruff and black use 88 by default, I think you can just remove the config

Reply:
done